<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual move</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMoveAllRight">
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/buttons/toright.gif</normaloff>:/buttons/toright.gif</iconset>
   </property>
   <property name="text">
    <string>Переместить все</string>
   </property>
   <property name="toolTip">
    <string>Переместить все файлы направо</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/buttons/refresh2.gif</normaloff>:/buttons/refresh2.gif</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить</string>
   </property>
  </action>
  <action name="actionMoveAllLeft">
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/buttons/toleft.gif</normaloff>:/buttons/toleft.gif</iconset>
   </property>
   <property name="text">
    <string>Все налево</string>
   </property>
   <property name="toolTip">
    <string>Переместить все файлы налево</string>
   </property>
  </action>
  <action name="actionMoveAll">
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/buttons/toboth.gif</normaloff>:/buttons/toboth.gif</iconset>
   </property>
   <property name="text">
    <string>Переместить все файлы</string>
   </property>
   <property name="toolTip">
    <string>Переместить все файлы</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resurs.qrc"/>
 </resources>
 <connections/>
</ui>
